datasets/us/dod_chinese_milcorps/crawler.py

```diff
@@ -0,0 +1,26 @@
+from zavod import Context
+import csv
+
+def crawl(context: Context):
+    source_file = context.fetch_resource("source.csv", context.dataset.data.url)
+    with open(source_file, "r") as f:
+        reader = csv.DictReader(f)
+        for row in reader:
+            entity = context.make("Company")
+            name = row.pop("Name")
+            entity.id = context.make_id(name)
+            entity.add("name", name)
+            entity.add("alias", row.pop("Alias", None))
+            clean_name = row.pop("Clean Name", None)
+            if clean_name != "" and clean_name != name:
+                entity.add("alias", clean_name)
```

Comment:
I think in this case you can ignore name, and simply always add clean_name to the name property, and alias to alias if set.